### DISCUSSION TITLE
Enable and display HTML in the FUD GUI

### DISCUSSION THREAD

**Discussion by GaryM0101** [fudforum/FUDforum#28] [Ideas]
FUD's GUI doesn't interpret/display HTML, I suspect this was a security concern because of the IRC and anonymous messages that could be hacked and posted. Forward 20 years later I believe today things have changed and we should have the option to interpret it wherever needed.

Business Case: Import Ebay RSS listings using the XML aggregator into FUD, interpret and display the HTML . 
Doing this today we see this in the FUD GUI:
![Curvex_html](https://user-images.githubusercontent.com/91424433/198736125-24608b13-ef6a-409a-8a90-c233d37e3acf.JPG)

We'd like the option to be able to turn on interpret and display the HTML where needed or a selection the XML aggregator or a plugin that we could enable:
![Curvex_html_ie](https://user-images.githubusercontent.com/91424433/198736241-6034e7c5-4cfb-4be1-9785-1f56ebda4dcc.JPG)

Thanks Frank and let us know your thoughts!
Gary
https://vintagegruen.org/

**Comment by naudefj** [comment]
Hi Gary,

I've never tested this before, but you can change the "Tag Style" of a forum from BBcode to HTML.

_Tag Style:
What markup language may users use. The tag style must match the content that will be stored in the form._

Another option (probably safer) may be to map a custom BBCode tag to your HTML via a plugin.

Best regards.
Frank

**Reply by GaryM0101** [reply]
Hi Frank, that worked, thanks for the suggestion!

![gp1](https://user-images.githubusercontent.com/91424433/198837111-ad555add-a15f-4a63-85b7-7c54c99620d1.JPG)
